torch/fx/experimental/proxy_tensor.py

```diff
@@ -652,6 +652,8 @@ def wrapped(*args):
         if tracing_mode == "symbolic":
             python_dispatcher_mode = enable_python_dispatcher()
 
+        constant_lifting_mode = _enable_constant_lifting() if _constant_lifting else nullcontext()
```

Comment:
constant_lifting is a bit of a misnomer at the moment. we don't have any guards in place to ensure that the arguments we lift are constant.

Reply:
Sure I can rename and put this under FakeTensorMode